pkg/reconciler/ingress/lister.go

```diff
@@ -73,7 +73,7 @@ func (l *gatewayPodTargetLister) getIngressUrls(ing *v1alpha1.Ingress, gatewayIp
 	for _, rule := range ing.Spec.Rules {
 		var target status.ProbeTarget
 
-		domains := getDomains(rule)
+		domains := rule.Hosts
```

Comment:
I am not confidence but it needs to be:

```
hosts := rule.Hosts
domain := hosts
```

I wonder that`domains := rule.Hosts` is problematic like https://golang.org/doc/faq#closures_and_goroutines

Reply:
not sure I parse that. Why would that be problematic :thinking:. If we mutated it, maybe, but then we'd have to clone the entire slice anyway.

Reply:
Ah, never mind. The `domains` is created inside the loop and no problem. Sorry for my stupidness :disappointed: